stable/postgresql/values.yaml

```diff
@@ -261,8 +266,11 @@ service:
 ## If this option is present and value is `true`,
 ## to the target database pod will be mounted a new tmpfs volume to remove
 ## this limitation.
+## If chmod.enabled is `true`, init container will chmod 777 `/dev/shm`.
 shmVolume:
   enabled: true
+  chmod:
+    enabled: true
 
 ## PostgreSQL data Persistent Volume Storage Class
 ## If defined, storageClassName: <storageClass>
```

Comment:
Please sync the `values.yaml` with the `values-production.yaml`

Reply:
/solved